src/pages/Staking/MyStakesTable.tsx

```diff
@@ -125,7 +141,7 @@ const MyStakesTable = () => {
 
     setActiveStakes(activeStakes);
     setPendingWithdrawals(pendingWithdrawals);
-  }, [delegateStakes, gateways, isFetching]);
+  }, [delegateStakes, gateways, isFetching, protocolBalance]);
```

Comment:
this feels like its worth a useQuery hook with the dependencies as keys

Reply:
I think it's alright here as it's not going to be re-used elsewhere. Perhaps there's an opportunity to further optimize but for now I think it's okay to keep this as-is.